en/flight_controller/mRo-X2.1.md

```diff
@@ -0,0 +1,72 @@
+# mRobotics-X2.1 Autopilot
+
+The [mRo](http://www.mRobotics.io/) *mRo-X2.1 autopilot* is based on the [Pixhawk-project](https://pixhawk.org/) **FMUv2** open hardware design. It runs PX4 on the [NuttX](http://nuttx.org) OS.
```

Comment:
It could be useful for users to understand how this compares to other autopilots. Typically comparisons are relative to original 3DR Pixhawk, or the mRO variant :-)

Reply:
It is basically the successor of the AUAV-X2 (mRo is the new distributor for AUAV Products since August 17). I'll note this on the AUAV-X2 page as well. I basically is a Pixhawk just smaller with a few improvements and newer Sensor generation.